CHANGES.md

```diff
@@ -0,0 +1,21 @@
+## v0.1.1
+
+Conduit 0.1.1 is focused on making it easier to get started with Conduit.
+
+* Conduit can now be installed on Kubernetes clusters that use RBAC
+* The new `--skip-outbound-ports` flag for the CLI `inject` command directs Conduit to bypass proxying for specific outbound ports, making Conduit easier to use with non-gRPC or HTTP/2 protocols.
```

Comment:
tioli nit: generally prefer to wrap lines at ~100 chars

Reply:
This is my preference too, but Github includes the line breaks when rendering markdown.

Compare, this list with no wrapping:

* Conduit can now be installed on Kubernetes clusters that use RBAC
* The new `--skip-outbound-ports` flag for the CLI `inject` command directs Conduit to bypass proxying for specific outbound ports, making Conduit easier to use with non-gRPC or HTTP/2 protocols.
* The CLI `tap` command output has been reformatted to be line-oriented, making it easier to parse with common UNIX command line utilities
* Proxies now support routing to non-fully qualified domain names, with routing rules that more closely resemble the default Kubernetes DNS search path
* The web UI has improved support for large deployments and deployments that don’t have any inbound/outbound traffic

To this one with wrapping at 100 characters:

* Conduit can now be installed on Kubernetes clusters that use RBAC
* The new `--skip-outbound-ports` flag for the CLI `inject` command directs Conduit to bypass
  proxying for specific outbound ports, making Conduit easier to use with non-gRPC or HTTP/2
  protocols.
* The CLI `tap` command output has been reformatted to be line-oriented, making it easier to parse
  with common UNIX command line utilities
* Proxies now support routing to non-fully qualified domain names, with routing rules that more
  closely resemble the default Kubernetes DNS search path
* The web UI has improved support for large deployments and deployments that don’t have any
  inbound/outbound traffic

Actually, on comparison, I think I liked the wrapped version better. Will change it.